### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by alt3ri**
Sorry Paimon, they forgor 💀

### CODE REVIEW

**Comment by lunaticwhat** [comment]
he forgor💀

**Comment by alt3ri** [comment]
I forgor too 💀

File: README.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ As well as setting up **`cheat-library`** as startup project.
 1. Run `injector.exe`.
 
 <h1 align="center">Features</h1>
+<h2>Press F1 to open the cheat menu first</h2>
```
Comment (lunaticwhat) [inline README.md:43]: `Press F1 to open Akebi GUI` or `Press F1 to open the menu` is enough

**Review by lunaticwhat — COMMENTED** [review]
a

File: README.md (hunk 1 above)
Comment (lunaticwhat) [reply README.md:43]: oh nvm. I just read the readme again and it shoud be a last step in **Usage** section instead of feature.

**Review by CPP-Callow — CHANGES_REQUESTED** [review]
Check @lunaticwhat review.

File: README.md [hunk 2]
```diff
@@ -40,6 +40,7 @@ As well as setting up **`cheat-library`** as startup project.
 1. Run `injector.exe`.
 
 <h1 align="center">Features</h1>
+<h2>Press F1 to open Akebi GUI</h2>
```
Comment (CPP-Callow) [inline README.md:43]: Bruh. Why you didn't move it to correct section?

**Review by CPP-Callow — CHANGES_REQUESTED** [review]

**Pull request merged**